.ci.yaml

```diff
@@ -434,52 +428,52 @@ targets:
       - packages/flutter_tools/lib/src/test/**
       - bin/**
 
-  - name: mac_gradle_plugin_bundle_test
+  - name: Mac gradle_plugin_bundle_test
     builder: Mac gradle_plugin_bundle_test
     scheduler: luci
     runIf:
       - dev/**
       - bin/**
 
-  - name: mac_gradle_plugin_fat_apk_test
+  - name: Mac gradle_plugin_fat_apk_test
     builder: Mac gradle_plugin_fat_apk_test
     scheduler: luci
     runIf:
       - dev/**
       - bin/**
 
-  - name: mac_gradle_plugin_light_apk_test
+  - name: Mac gradle_plugin_light_apk_test
     builder: Mac gradle_plugin_light_apk_test
     scheduler: luci
     runIf:
       - dev/**
       - bin/**
 
-  - name: mac_module_custom_host_app_name_test
+  - name: Mac module_custom_host_app_name_test
     builder: Mac module_custom_host_app_name_test
     scheduler: luci
     runIf:
       - dev/**
       - packages/flutter_tools/**
       - bin/**
 
-  - name: mac_module_host_with_custom_build_test
+  - name: Mac module_host_with_custom_build_test
     builder: Mac module_host_with_custom_build_test
     scheduler: luci
     runIf:
       - dev/**
       - packages/flutter_tools/**
       - bin/**
 
-  - name: mac_module_test
+  - name: Mac module_test
     builder: Mac module_test
     scheduler: luci
     runIf:
       - dev/**
       - packages/flutter_tools/**
       - bin/**
 
-  - name: mac_module_test_ios
+  - name: Mac module_test_ios
```

Comment:
I'm not sure how to prevent these breakages, other than as a human process where I ensure my checkout is up to date before merging.